internal/dao/pgx/account_pgx.go

```diff
@@ -76,6 +77,15 @@ func (x *accountDao) GetOrCreateByIdentity(ctx context.Context, orgId string, ac
 		return nil, fmt.Errorf("pgx error: %w", err)
 	}
 
+	// Step 4: requery in case transaction isolation (simultaneous transactions)
+	if result.ID == 0 && result.OrgID == "" {
```

Comment:
Query again makes sense (it will increase the chances of success :) ), but will we ever trigger this, given we return on error few lines before?